bench-tool/bench-tool.cabal

```diff
@@ -1,10 +1,10 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.34.6.
+-- This file has been generated from package.yaml by hpack version 0.35.0.
```

Comment:
It's not on purpose, it was what was generated using the stack version speciefied in lts-20.4, i.e. Stack @ 2.9.1. I think it makes sense to regenerate the file since the dependencies were bumped, but is the line in question problematic?